bitcoin/src/blockdata/script/witness_program.rs

```diff
@@ -99,7 +115,10 @@ impl WitnessProgram {
     pub fn version(&self) -> WitnessVersion { self.version }
 
     /// Returns the witness program.
-    pub fn program(&self) -> &PushBytes { &self.program }
+    pub fn program(&self) -> &PushBytes {
+        // Always succeeds
+        self.program.as_slice().try_into().unwrap()
```

Comment:
We tend to use `expect` with a message why it won't panic instead of a code comment, for example
```rust
        self.program.as_slice().try_into().expect("slice length is always valid")
```

Reply:
Fixed :white_check_mark: